karma.conf.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+module.exports = function(config) {
+
+  var ramdaMain = './node_modules/ramda/dist/ramda.js';
+  var sanctuaryDefMain = './node_modules/sanctuary-def/index.js';
+
+  // https://saucelabs.com/platforms
+  var customLaunchers = {
```

Comment:
Since this doesn't reference `config` let's move it to the top level of the file. In fact, since we only reference `config` in one place I'd like to see `module.exports` reduced to a one-liner:

```javascript
module.exports = function(config) { config.set(options); };
```

By defining `options` at the top level we'd be able to replace a 100-line impure function with a one-line impure function. :)